server/apiserver/argoserver.go

```diff
@@ -246,6 +257,7 @@ func (as *argoServer) newGRPCServer(instanceIDService instanceid.Service, offloa
 
 	grpcServer := grpc.NewServer(sOpts...)
 
+	log.Infof("GRPC Server Max Message Size is: %d", MaxGRPCMessageSize)
```

Comment:
no need to log here, log in the init function

Reply:
also, please use structured logging

Reply:
ok, followed that as well